eclass/perl-module.eclass

```diff
@@ -252,21 +224,6 @@ perl-module_src_configure() {
 
 	if [[ ( ${PREFER_BUILDPL} == yes || ! -f Makefile.PL ) && -f Build.PL ]] ; then
 		case ${EAPI} in
-			6)
-				if grep -q '\(use\|require\)\s*Module::Build::Tiny' Build.PL ; then
-					einfo "Using Module::Build::Tiny"
-					if [[ ${DEPEND} != *dev-perl/Module-Build-Tiny* && ${PN} != Module-Build-Tiny ]]; then
-						eerror "QA Notice: The ebuild uses Module::Build::Tiny but doesn't depend on it."
-						die    " Add dev-perl/Module-Build-Tiny to DEPEND!"
-					fi
-				else
-					einfo "Using Module::Build"
-					if [[ ${DEPEND} != *virtual/perl-Module-Build* && ${DEPEND} != *dev-perl/Module-Build* && ${PN} != Module-Build ]] ; then
-						eerror "QA Notice: The ebuild uses Module::Build but doesn't depend on it."
-						die    " Add dev-perl/Module-Build to DEPEND!"
-					fi
-				fi
-				;;
 			*)
 				if grep -q '\(use\|require\)\s*Module::Build::Tiny' Build.PL ; then
 					einfo "Using Module::Build::Tiny"
```

Comment:
another edo candidate on 249

Reply:
you want to do `edo` on
```
einfo "perl [Build.PL](http://build.pl/)" "$@"
		perl [Build.PL](http://build.pl/) "$@" <<< "${pm_echovar}" \
				|| die "Unable to build!"
```
? sounds a bit audacious :grinning:

Reply:
❌